client/me/get-apps/mobile-download-card.jsx

```diff
@@ -6,46 +6,263 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
 import { localize } from 'i18n-calypso';
-import { identity } from 'lodash';
 
 /**
  * Internal dependencies
  */
-import Card from 'components/card';
 import AppsBadge from 'me/get-apps/apps-badge';
+import Card from 'components/card';
+import Button from 'components/button';
+import QuerySmsCountries from 'components/data/query-countries/sms';
+import QueryAccountRecoverySettings from 'components/data/query-account-recovery-settings';
+import QueryUserSettings from 'components/data/query-user-settings';
+import FormPhoneInput from 'components/forms/form-phone-input';
+import getCountries from 'state/selectors/get-countries';
+import { successNotice, errorNotice } from 'state/notices/actions';
+import {
+	getAccountRecoveryPhone,
+	isAccountRecoverySettingsReady,
+} from 'state/account-recovery/settings/selectors';
+import getUserSettings from 'state/selectors/get-user-settings';
+import hasUserSettings from 'state/selectors/has-user-settings';
+import { sendSMS } from 'state/mobile-download-sms/actions';
+import phoneValidation from 'lib/phone-validation';
+import config from 'config';
+import userAgent from 'lib/user-agent';
+
+import {
+	isAppSMSRequestSending,
+	didAppSMSRequestCompleteSuccessfully,
+	didAppSMSRequestCompleteWithError,
+} from 'state/selectors/get-apps-sms-request';
+
+class MobileDownloadCard extends React.Component {
+	static displayName = 'SecurityAccountRecoveryRecoveryPhoneEdit';
```

Comment:
Mostly because I copied this from `SecurityAccountRecoveryRecoveryPhoneEdit ` and forgot to update the `displayName` 🙊.

Fixed in 1170498699247e24e7d5b5d94ee9682d4f69596d – I'm believe we want to specify one in order to have nice error messaging if this card does something wrong?

Reply:
I believe it's already part of `@babel/react` preset and it will use the name of the Function by default. It's useful when we want to customize it for instance in HOCs.